From fd6d82ffaedc32abdba1d9c9e45a96ba8f9c8963 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Thu, 30 Aug 2018 06:51:19 -0500
Subject: [PATCH 8021/8021] misc: intel: Makefile and Kconfig

---
 drivers/misc/Kconfig  | 3 +++
 drivers/misc/Makefile | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/drivers/misc/Kconfig b/drivers/misc/Kconfig
index 8136dc7e863d..184f0f17105d 100644
--- a/drivers/misc/Kconfig
+++ b/drivers/misc/Kconfig
@@ -513,6 +513,9 @@ source "drivers/misc/ti-st/Kconfig"
 source "drivers/misc/lis3lv02d/Kconfig"
 source "drivers/misc/altera-stapl/Kconfig"
 source "drivers/misc/mei/Kconfig"
+source "drivers/misc/intel/sepdk/Kconfig"
+source "drivers/misc/intel/soc_perf/Kconfig"
+source "drivers/misc/intel/socwatch/Kconfig"
 source "drivers/misc/vmw_vmci/Kconfig"
 source "drivers/misc/mic/Kconfig"
 source "drivers/misc/genwqe/Kconfig"
diff --git a/drivers/misc/Makefile b/drivers/misc/Makefile
index ad0e64fdba34..0522ec3c1420 100644
--- a/drivers/misc/Makefile
+++ b/drivers/misc/Makefile
@@ -44,6 +44,9 @@ obj-y				+= lis3lv02d/
 obj-$(CONFIG_USB_SWITCH_FSA9480) += fsa9480.o
 obj-$(CONFIG_ALTERA_STAPL)	+=altera-stapl/
 obj-$(CONFIG_INTEL_MEI)		+= mei/
+obj-$(CONFIG_INTEL_SEP)		+= intel/sepdk/
+obj-$(CONFIG_INTEL_SOCPERF)	+= intel/soc_perf/
+obj-$(CONFIG_INTEL_SOCWATCH)	+= intel/socwatch/
 obj-$(CONFIG_VMWARE_VMCI)	+= vmw_vmci/
 obj-$(CONFIG_LATTICE_ECP3_CONFIG)	+= lattice-ecp3-config.o
 obj-$(CONFIG_SRAM)		+= sram.o
-- 
2.18.0

